templates/base.html

```diff
@@ -18,23 +17,26 @@
 <div class="container-fluid">
   <div class="row">
     <div class="col-sm-2">
-
       {% block sidebar %}
         {% include "includes/sidebar.html" %}
       {% endblock %}
-
+      <!-- Display username and login/logout links in sidebar -->
+      <div class="user-auth">
+        {% if user.is_authenticated %}
+          <p>Welcome, <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a>!</p>
+          <a href="{% url 'logout' %}">Logout</a>
+        {% else %}
+          <a href="{% url 'login' %}">Login</a>
```

Comment:
The URL tag `{% url 'login' %}` should match the name of the login path defined in your `urls.py`. Ensure consistency between the template and URL configuration to avoid navigation issues.